docs/vm/resource-overcommit.md

```diff
@@ -58,3 +58,13 @@ To address the issue, Harvester provides an annotation `harvesterhci.io/reserved
 ## Why my virtual machines are scheduled unevenly?
 
 The scheduling of virtual machines depends on the underlying behavior of the kube-scheduler. We have a dedicated article explaining the details. If you would like to learn more, check out:  [Harvester Knowledge Base: VM Scheduling](https://harvesterhci.io/kb/vm-scheduling/).
+
+## Insufficient resource quota
+
+When creating or starting multiple VMs simultaneoysly, the values configured in `overcommit-config` may cause the `ResourceQuota` to be less than the resources used for the batch of VMs. As a result, you will see the following error message:
+
+![insufficient-resource-quota-error](/img/v1.2/vm/insufficient-resource-quota-error.png)
+
+The annotation `harvesterhci.io/insufficient-resource-quota: xxxxx` has been added to the VirtualMachine custom resource to let you know the VM was shutdown due to insufficient resources.
+
+![insufficient-resource-quota-annotation](/img/v1.2/vm/insufficient-resource-quota-annotation.png)
```

Comment:
Overcommit doesn't affect ResourceQuota, and here the insufficient annotation has nothing to do with overcommit and should be deleted.
I have created a new PR for the ResourceQuota document that also contains the content here, thanks.